README.md

```diff
@@ -1643,6 +1751,11 @@ Server: spray-can/1.3.3
 }
 ```
 
+## GET /api/workflows/:version/:id/timing
+
+This endpoint is meant to be used in a web browser.  It will show a Gantt Chart of a particular WDL workflow.  The bars in the chart represent start and end times for individual task invocations.
+
+![Timing diagram](https://s3.amazonaws.com/uploads.hipchat.com/23227/2330742/3W5kmbu6oYAzyW8/NHLBI%20Long%20Tail.png)
```

Comment:
Make this image much smaller, if you want people to be able to access the full size provide a small version which clicks through to the full one